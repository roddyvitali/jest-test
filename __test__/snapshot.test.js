import { getCharacter } from '../snapshot';
import rick from '../rick.json';

describe('Es hora de las intantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });

    test('Siempre fallara la instantanea', () => {
        const user = {
            createAt: new Date(),
            id: Math.floor(Math.random() * 20),
        }
        expect(user).toMatchSnapshot();
    });

    test('Tenemos una excepciÃ³n dentro del codigo', () => {
        const user = {
            id: Math.floor(Math.random() * 20),
            name: "Roddy Vitali Astorga"
        }
        expect(user).toMatchSnapshot({
            id:  expect.any(Number)
        });
    });

});