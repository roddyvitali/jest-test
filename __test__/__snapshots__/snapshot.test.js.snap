// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las intantaneas Siempre fallara la instantanea 1`] = `
Object {
  "createAt": 2018-10-29T03:32:40.834Z,
  "id": 10,
}
`;

exports[`Es hora de las intantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las intantaneas Tenemos una excepci√≥n dentro del codigo 1`] = `
Object {
  "id": Any<Number>,
  "name": "Roddy Vitali Astorga",
}
`;
